spec/services/dttp/contact_update_spec.rb

```diff
@@ -21,7 +21,7 @@ module Dttp
       let(:placement_payload) { Params::PlacementAssignment.new(trainee).to_json }
 
       before do
-        enable_features(:persist_to_dttp, "routes.school_direct_salaried")
+        enable_features(:persist_to_dttp, "routes.school_direct_salaried", "show_funding", "send_funding_to_dttp")
```

Comment:
```suggestion
enable_features(:persist_to_dttp, "routes.school_direct_salaried", :show_funding, :send_funding_to_dttp)
```

Just for consistency :)